db/migrate/20160830233558_migrate_ports_to_port_ranges_in_load_balancer_listener.rb

```diff
@@ -0,0 +1,26 @@
+class MigratePortsToPortRangesInLoadBalancerListener < ActiveRecord::Migration[5.0]
+  class LoadBalancerListener < ActiveRecord::Base
+  end
+
+  def up
+    LoadBalancerListener.where.not(:instance_port => nil).each do |x|
+      x.update(:instance_port_range => x.instance_port..x.instance_port,
```

Comment:
It is.  This is to migrate any existing values (where we only supported a single port forward).